src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private Node<K, V>[] nodes;
+    private int size;
+    private double threshold;
+
+    public MyHashMap() {
+        nodes = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        threshold = LOAD_FACTOR * DEFAULT_CAPACITY;
+        size = 0;
```

Comment:
0 is the default value for int type, so we can remove initialization, if you want